pkg/machines/vmdiskstab.jsx

```diff
@@ -120,22 +120,19 @@ const VmDisksTab = ({ vm, provider }) => {
         supported, but not available yet ~ will be read soon
         unsupported ~ failed attempt to read (old libvirt)
      */
-    let areDiskStatsSupported = true;
-    let showDiskStatsColumns = false;
+    let areDiskStatsSupported = false;
     if (vm.disksStats) {
-        const isNaNFound = Object.getOwnPropertyNames(vm.disksStats).some(target => { // look for NaN value
+        const isNonNaNFound = Object.getOwnPropertyNames(vm.disksStats).some(target => { // look for non-NaN value (means disk stats at least partially read)
```

Comment:
how about renaming this to something like `hasStatistics` and comment `// we have stats if there is at least one non-NaN value`
"non-NaN" is a double negative